draft-jasdips-regext-rdap-rpki.md

```diff
@@ -686,6 +696,230 @@ Here is an elided example for an autonomous system number object with ASPAs:
+
+The BGPSec Router Certificate object class can contain the following members:
+
+* objectClassName -- the string "rpki bgpsec router cert"
+* handle -- a string representing the RIR-unique identifier of the BGPSec Router Certificate registration
+* serialNumber -- a string representing the unique identifier for the certificate
+* issuer -- a string representing the Certificate Authority (CA) that issued the certificate
+* signatureAlgorithm -- a string representing the algorithm used by the CA to sign the certificate
+* subject -- a string representing the identity of the router
+* subjectPublicKeyInfo -- an object representing the subject's public key information ([@!RFC8208, section 3.1]), with
+  the following members:
+    * publicKeyAlgorithm -- a string representing the algorithm for the public key
+    * publicKey -- a string representation of the public key
+* autnum -- an unsigned 32-bit integer representing the autonomous system number that the router emits secure route
+  advertisements on behalf of ([@!RFC8209, section 3.1.3.5])
+* notValidBefore -- a string that contains the time and date in Zulu (Z) format with UTC offset of 00:00 ([@!RFC3339]),
+  representing the not-valid-before date of the certificate
+* notValidAfter -- a string that contains the time and date in Zulu (Z) format with UTC offset of 00:00 ([@!RFC3339]),
+  representing the not-valid-after date of the certificate
+* events -- see [@!RFC9083, section 4.5]
+* links -- links ([@!RFC9083, section 4.2]) for "self", and "related" to autonomous system number and IRR (when defined)
+  objects
+* remarks -- see [@!RFC9083, section 4.3]
+
+Here is an elided example of a BGPSec Router Certificate object in RDAP:
+
+```
+{
+  "objectClassName": "rpki bgpsec router cert",
+  "handle": "ABCD",
+  "serialNumber": "1234",
+  "issuer": "CN=ISP-CA",
+  "signatureAlgorithm": "ecdsa-with-SHA256",
+  "subject": "CN=ROUTER-ASN-65536",
+  "subjectPublicKeyInfo":
+  {
+    "publicKeyAlgorithm": "id-ecPublicKey",
+    "publicKey": "...",
+  }
+  "autnum": 65536,
+  "notValidBefore": "2024-04-27T23:59:59Z",
+  "notValidAfter": "2025-04-27T23:59:59Z"
+  "events":
+  [
+    {
+      "eventAction": "registration",
+      "eventDate": "2024-01-01T23:59:59Z"
+    },
+    ...
+  ],
+  "links":
+  [
+    {
+      "value": "https://example.net/rdap/rpkiBgpsecRouterCert/65536",
+      "rel": "self",
+      "href": "https://example.net/rdap/rpkiBgpsecRouterCert/65536",
+      "type": "application/rdap+json"
+    },
+    {
+      "value": "https://example.net/rdap/rpkiBgpsecRouterCert/65536",
+      "rel": "related",
+      "href": "https://example.net/rdap/autnum/65536",
+      "type": "application/rdap+json"
+    },
+    ...
+  ],
+  "remarks":
+  [
+    {
+      "description": [ "A BGPSec Router Certificate object in RDAP" ]
+    }
+  ]
+}
+```
+
+## Lookup
+
+The resource type path segment for exact match lookup of a BGPSec Router Certificate object is "rpkiBgpsecRouterCert".
```

Comment:
See the latest additions to the extensions draft and how RFC 9082 addresses this... the path should either be "rpki1_bgpsecRouterCert" or "rpki1/bgpsecRouterCert".

personally I think the camel casing gets annoying and snake case is easier to read so "rpki1/bgpsec_router_cert" is how I find it easier to read.

Reply:
> See the latest additions to the extensions draft and how RFC 9082 addresses this... the path should either be "rpki1_bgpsecRouterCert" or "rpki1/bgpsecRouterCert".
> 
> personally I think the camel casing gets annoying and snake case is easier to read so "rpki1/bgpsec_router_cert" is how I find it easier to read.

Good point. Will change here and for other object classes.